### PR TITLE
Bug fix in TRD hits

### PR DESCRIPTION
**Pull request by jolopezl**
This PR fixes a bug when reading a keyword to identify sensitive regions for the TRD hit creation.

### CODE REVIEW

File: Detectors/TRD/simulation/src/Detector.cxx [hunk 1]
```diff
@@ -121,10 +121,11 @@ bool Detector::ProcessHits(FairVolume* v)
   }
   // Calculate the charge according to GEANT Edep
   // Create a new dEdx hit
-  const double enDep = fMC->Edep();
+  const double enDep = TMath::Max(fMC->Edep(), 0.0) * 1.0e+9;
   // Store those hits with enDep bigger than the ionization potential of the gas mixture for in-flight tracks
   // or store hits of tracks that are entering or exiting
-  if ((enDep > mWion) || trkStat) {
+  int qTot = (int)(enDep / mWion);
+  if (qTot || trkStat) {
```
Comment (sawenzel) [inline Detectors/TRD/simulation/src/Detector.cxx]: to me the old condition looks the same (and more efficient since you avoid a division). Is this change necessary?

Comment (jolopezl) [reply Detectors/TRD/simulation/src/Detector.cxx]: I agree with you. I was using this for debugging purposes only.
